site/content/contribute/server/developer-setup/windows-wsl.md

```diff
@@ -22,8 +22,18 @@ This is an unofficial guide. Community testing, feedback and improvements are we
     ```
     If you don't migrate your data, the new, docker-compose-managed containers will start out empty. To remove the old containers -- destroying any existing data -- use `make clean-old-docker`.
     
+2. Install docker-compose (using bash)
 
-4. Install Go (using bash):
+    ```sh
+    sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    sudo chmod +x /usr/local/bin/docker-compose
+    ```
+
+4. Install docker-compose
+   
+   You should install docker-compose
```

Comment:
```suggestion
   You should install docker-compose.
```

Reply:
Wait a minute, the entire step 4 isn't supposed to be here - that was simply a draft which I replaced (or at least I thought so) later with the real version.
I added a new commit to this PR which removes this old artifact and properly numbers the docker-compose installation step.

Sorry for the inconvenience :/